CONTRIBUTING.md

```
@@ -1,335 +1,557 @@
 
-- Uma sequência grande de letras e números vai aparecer. Clique no botão para copiar essa chave e guarde-a (não feche a página antes de copiar!).
+O `-b` cria a branch e já muda para ela.
 
+### 3. Trabalhe e faça commits
 
-2. Configurando suas Variáveis de Ambiente
-O projeto precisa saber qual é a sua chave para conseguir conversar com o Gemini.
+Use mensagens no padrão [Conventional Commits](https://www.conventionalcommits.org/pt-br/):
 
-- Abra o seu arquivo `.env.local` (que você criou seguindo o passo a passo do [README](./README.md#-4-configure-o-arquivo-env)).
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)